sorc/ncep_post.fd/MISCLN.f

```diff
@@ -2241,7 +2241,7 @@ SUBROUTINE MISCLN
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
-                   IF(T1D(I,J) < spval) GRID1(I,J) = EGRID1(I,J)
+                   IF(TBND(I,J,1) < spval) GRID1(I,J) = EGRID1(I,J)
```

Comment:
EGRID1 should be checked.

Reply:
grid1 is initialized with spval. so if (egrid2==spval) grid1=egrid2 still valid.